content/services/hosting/projects.csv

```diff
@@ -235,7 +235,7 @@
 "Software Freedom Conservatory","vm"
 "Spack", "aarch64, powerpc"
 "Strimzi","powerpc"
-"sourceware","aarch64"
+"Sourceware","aarch64, vm"
```

Comment:
It looks like I never added `co-location` when I set up their machine. Can you please also add this?
```suggestion
"Sourceware","aarch64, vm, co-location"
```

Reply:
10-4, will do.